### DISCUSSION TITLE
New termwiz release?

### DISCUSSION THREAD

**Discussion by quark-zju** [wezterm/wezterm#953] [General]
We are looking for a new streampager release addressing some of the internal feature requests. It seems termwiz has got some fixes. Perhaps it's worth a new release too?

**Comment by wez** [comment]
There have been a couple of breaking changes, and I don't think any of the changes since the last release are helpful for streampager; they're mostly (all?) for wezterm.  If there aren't any things that you really need I'd prefer to defer making a new termwiz release right now!

**Reply by quark-zju** [reply]
Thanks! Will just upgrade streampager now.
